backend/job_board/models.py

```diff
@@ -0,0 +1,48 @@
+import json
+from copy import copy
+
+from django.conf import settings
+from django.db import models
+from django.db.models import Q
+from django.utils.text import slugify
+from django.utils.translation import gettext_lazy as _
+from i18n.fields import I18nCharField, I18nTextField
+from model_utils.models import TimeStampedModel
+
+
+class JobListingManager(models.Manager):
+    def by_slug(self, slug):
+        term = json.dumps(slug)
+
+        filters = Q()
+
+        for lang, __ in settings.LANGUAGES:
+            filters |= Q(**{f"slug__{lang}": term})
+
+        return self.get_queryset().filter(filters)
+
+
+class JobListing(TimeStampedModel):
```

Comment:
should we add OrderedModel so we can order the job listings?

Reply:
I think they should be ordered by creation date, we could add some additional flags like "highlighted" to move them at the top if needed, but we don't need custom order for now :)

Reply:
So we won't have cases where a sponsor who paid 5k wants their job to appear at the start and not below those who paid 500 euros or whatever?

Fine if we don't want to add it for now